src/i18n/en.ts

```diff
@@ -142,8 +142,9 @@ export default {
 		"ROW": "row"
 	},
 	"TABLE_TOOLBAR": {
-		"ACTION_BAR": "Table Action Bar",
-		"BATCH_TEXT": "items selected",
+		"ACTION_BAR": "Table action bar",
+		"BATCH_TEXT_SINGLE": "1 item selected",
```

Comment:
So we don't break anyone this should just be `BATCH_TEXT` 👍

Reply:
To make sure I understand correctly, you mean to use `"BATCH_TEXT": "1 item selected"` here? I think it would still break since the string has changed? Or should we keep the old string `"BATCH_TEXT": "items selected"` as-is, but don't reference it in code?

Reply:
The string can change, my primary concern is users that have customized the text suddenly not seeing it ... We may actually be best to leave `BATCH_TEXT_SINGLE` as it was, and if we don't have `BATCH_TEXT_SINGLE` and `BATCH_TEXT_MULTIPLE` we just switch back to `BATCH_TEXT` ... that way people can adopt the new translation keys, and the old way doesn't break anyone

Reply:
OK, given how `I18n` class works, it means `BATCH_TEXT_SINGLE` and `BATCH_TEXT_MULTIPLE` will always be available and the English version will be the fallback. So we can't do the check you mentioned.

Alternatively, maybe we can check `BATCH_TEXT`, and it is NOT `items selected` then we use it, otherwise we'll use `BATCH_TEXT_SINGLE` and `BATCH_TEXT_MULTIPLE`? This would preserve `BATCH_TEXT` in case someone customized it...

Reply:
Hmm ... What we _could_ do, is remove/null `BATCH_TEXT` in the shipped `EN.ts`, and check for it's existence - anyone consuming the library without modifications will get the new (better) translations, and if `BATCH_TEXT` is set we can assume that was intentional

Reply:
OK, sounds good

Reply:
@cal-smith so I set `BATCH_TEXT` to an empty string in en.ts, and found that I had to fix `getValueFromPath` since it will throw an exception for empty string (which doesn't make sense).

Reply:
Interesting 🤔 but great work getting that fixed!